src/api_server/src/lib.rs

```diff
@@ -707,11 +655,11 @@ mod tests {
             .unwrap();
 
         // Wait for the server to set itself up.
-        thread::sleep(Duration::new(0, 10_000_000));
+        thread::sleep(Duration::from_millis(10));
         let mut sock = UnixStream::connect(PathBuf::from(path_to_socket)).unwrap();
 
         // Send a GET instance-info request.
-        assert!(sock.write_all(b"GET / HTTP/1.1\r\n\r\n").is_ok());
+        assert!(sock.write_all(b"GET /mmds HTTP/1.1\r\n\r\n").is_ok());
```

Comment:
is this a leftover?

Reply:
no, this is on purpose. This test verifies `bind_and_run` functionality so it uses the simplest API endpoint - which is now mmds.

URL `/` is served by Vmm, so to continue using that in this test, we would have to also simulate that part of the communication which is out of scope for this test.

Reply:
makes sense, let's update the comment then